Main/src/main/java/com/main/acad/controller/Controller.java

```diff
@@ -0,0 +1,39 @@
+package com.main.acad.controller;
+
+import com.main.acad.annotation.Mapping;
+import com.main.acad.annotation.Mapping.MappingMethod;
+import com.main.acad.dao.ChapterDao;
+import com.main.acad.dao.Dao;
+import com.main.acad.entity.Chapter;
+import com.main.acad.serializator.MyJsonSerializer;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+@Mapping(url = "/api")
```

Comment:
According to you annotation processing you don't need this annotation (you have under the methods)